azext_iot/central/providers/monitor_provider.py

```diff
@@ -0,0 +1,113 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.util import CLIError
+
+from azext_iot.central.providers import CentralDeviceProvider
+from azext_iot.common.utility import init_monitoring
+
+
+class MonitorProvider:
```

Comment:
Are you intending for this PR to also have vanilla IoT Hub monitoring able to use MonitorProvider directly? Just wondering since there are some central specific things here.

Reply:
No, I plan to have this monitor provider just be used by central. It should be relatively easy to replicate this pattern in IoT hub once iothub code is moved out of `operations.py`